tools/run_in_docker

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd $(dirname $0)/..
+docker run -v $(pwd):/usr/src/app/ -v $(pwd)/disk-cache:/disk-cache/ alxrsngrtn/arcs-travis-build:latest  /bin/sh -c "cd /usr/src/app; $1"
```

Comment:
Nit: delete extra space, and end file with newline.

Might also suggest splitting args over multiple lines for readability, but up to you.